src/pages/de/gaston-vs-gäste-rufsystem.md

```diff
@@ -0,0 +1,36 @@
+---
+lang: de
+title: Wie kann Gaston ein Gäste-Rufsystem ersetzen?
+description: Elektronische Rufsysteme gehören für viele gastronomische Betriebe längst zum Alltag. Der praktische Nutzen im Arbeitsalltag macht ein solches System für die moderne Gastronomie unverzichtbar.
+image: icon.png
+keyword: gäste-rufsystem
+layout: '@layouts/LandingPage.astro'
+---
+
+Das Konzept des Gäste-Rufsystems existiert seit Jahren und ist daher vielleicht die erste Idee, die einem in den Sinn kommt, wenn man an eine Erhöhung der Qualität im Service denkt. Aber mittlerweile existieren bessere Alternativen. Aus diesem Grund werden wir hier in nur vier Punkten argumentieren, warum Sie Gaston App anstatt eines Gäste-Rufsystems in Betracht ziehen sollten.
+
+## 1. Initialkosten
+
+Die Kosten für ein Gäste-Rufsystem belaufen sich auf bis zu 800 CHF.
```

Comment:
Woher kommt die Zahl 800 CHF? Ist dies für ein Geräte oder für mehrere Geräte?

Reply:
Ich habe bei Google Shopping die Preise mir angeschaut. So zwischen 300-800 (Set)

Reply:
OK, ich glaub in dem Fall ist dies kein so guter Grund. :) Habe nun stattdessen etwas über die Prozessoptimierung geschrieben. Was denkst du?